pkg/kanctl/actionset.go

```diff
@@ -92,6 +94,7 @@ func newActionSetCmd() *cobra.Command {
 	cmd.Flags().StringSliceP(deploymentFlagName, "d", []string{}, "deployment for the action set, comma separated namespace/name pairs (eg: --deployment namespace1/name1,namespace2/name2)")
 	cmd.Flags().StringSliceP(optionsFlagName, "o", []string{}, "specify options for the action set, comma separated key=value pairs (eg: --options key1=value1,key2=value2)")
 	cmd.Flags().StringP(profileFlagName, "p", "", "profile for the action set")
+	cmd.Flags().StringP(repositoryServerFlagName, "r", "", "kopia repository server custom resource reference (eg: --repository-server namespace/name)")
```

Comment:
After looking at the rest of the code it looks like `--repository-server name` is also allowed. In that case do we want  expect the repo server to be available in the controller namespace or in the app namespace. 
Also, instead of supporting both the formats we should just support `namespace/name` format, just to avoid confusion.

Reply:
Yes @viveksinghggits, it is allowed but we want the repository server CR in controller namespace only